src/omotes_orchestrator/db_models/job.py

```diff
@@ -49,5 +49,5 @@ class JobDB(Base):
     """Time at which the job is submitted to Celery."""
     running_at: Optional[datetime] = db.Column(db.DateTime(timezone=True))
     """Time at which a Celery worker has started the task for this job."""
-    timeout_after_ms: int = db.Column(db.Integer)
+    timeout_after_ms: Optional[int] = db.Column(db.Integer)
```

Comment:
python 3.10+you can use `int | None`

Reply:
Hey Mark, as discussed, since there are more places in the current code implementing this way of type hint, will stick to this for now.